bundles/statistics/statsgrid2016/view/SearchFlyout.js

```diff
@@ -351,22 +351,48 @@ Oskari.clazz.define('Oskari.statistics.statsgrid.view.SearchFlyout', function (t
         }
 
         // Run the searches to see if we get data from the service.
-        searchValues.forEach(search => {
-            const {datasource, indicator, selections, series, regionset} = search;
-            this.service.getIndicatorData(datasource, indicator, selections, series, regionset, (err, data) => {
-                if (err || !data) {
-                    searchFailed(search);
-                    return;
-                }
-                let counter = 0;
-                const enoughData = !!Object.values(data).find(val => !isNaN(val) && ++counter > 1);
-                if (!enoughData) {
-                    searchFailed(search);
-                    return;
-                }
-                searchSuccessfull(search);
-            });
+        const batchSize = 2;
+        let batchCounter = 0;
```

Comment:
The variable `batchCounter` keeps track of received results, but the name implies it counts batches. Maybe rename?